Runtime/Game/Requests/PurchaseRequest.cs

```diff
@@ -77,12 +81,68 @@ public class LootLockerPurchaseCatalogItemRequest
          /// </summary>
         public LootLockerCatalogItemAndQuantityPair[] items { get; set; }
     }
+
+    /// <summary>
+    /// 
+    /// </summary>
+    public class LootLockerRedeemAppleAppStorePurchaseForPlayerRequest
+    {
+#if LOOTLOCKER_SANDBOX_APPLE_APP_STORE
```

Comment:
Would it be possible to do this in a similar way as the Unreal SDK with a bool instead?
This requires a different workflow to change and will be somethinfg that we need to document, so if it's not too mcu hwork it would be better to not need to specify any scripting define symbols.

Reply:
Fixed